apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/wpcom-nux.js

```diff
@@ -108,9 +108,9 @@ function getWpcomNuxPages( isGutenboarding ) {
 			alignBottom: true,
 		},
 		{
-			heading: __( 'Private until you’re ready', 'full-site-editing' ),
+			heading: __( 'Coming Soon', 'full-site-editing' ),
 			description: __(
-				'Your site will remain private as you make changes until you’re ready to launch and share with the world.',
+				'Your site will remain visible only to you until you’re ready to launch and share it with the world.',
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.

Reply:
We should probably update the copy to mention the launch button in https://github.com/Automattic/wp-calypso/pull/46817

 **_Your site will remain visible only to you until you’re ready. Click "Launch" in the toolbar to share it with the world._**


Context: p1603350819431300-slack-C9EJ7KSGH